frontend/public/components/pod.tsx

```diff
@@ -839,9 +840,9 @@ export const PodList: React.FC<PodListProps> = withUserSettingsCompatibility<
 });
 PodList.displayName = 'PodList';
 
-export const filters = [
+export const filters = () => [
```

Comment:
Let's make it more obvious this is a function with the `get` prefix.

```suggestion
export const getFilters = () => [
```